src/stories/tabs/Tabs.tsx

```diff
@@ -25,10 +25,22 @@ export type TabsProps = {
      */
     tabWidth?: CSSLength;
     /**
-     * Stylistic alternates for the tabs. `fixed` will rely on the `tabWidth` prop and default to 150px; `compact` will use an alternate intrinsically-sized style that ignores the `tabWidth` prop; `auto` will use fixed style on larger screens and switch to compact style on smaller screens (below 640px, the default Paris theme's medium breakpoint).
+     * Stylistic alternates for the tabs.
+     * `fixed` will rely on the `tabWidth` prop and default to 150px;
+     * `compact` will use an alternate intrinsically-sized style that ignores the `tabWidth` prop;
+     * `auto` will use fixed style on larger screens and switch to compact style on smaller screens (below 640px, the default Paris theme's medium breakpoint).
+     * `full` will use a full-width style that ignores the `tabWidth` prop, with a taller height as well
      * @default auto
      */
-    kind?: 'fixed' | 'compact' | 'auto';
+    kind?: 'fixed' | 'compact' | 'auto' | 'full';
+    /**
+     * The thickness of the bottom bar. Defaults to `thick`.
+     */
+    barStyle?: 'thick' | 'thin';
+    /**
+     * The background style, with `default` being transparent and `glass` being a glassmorphism effect. Defaults to `default`.
+     */
+    backgroundStyle?: 'default' | 'glass';
```

Comment:
display issue with glass on firefox + chrome

![SCshot@2024-09-13_04-15-39PM@2x](https://github.com/user-attachments/assets/b682a190-12c7-4e43-8850-d163d225af8a)

Reply:
Okay so there were two issues in this image.

Issue 1 I just fixed, which was the tab bar extending past the container edge on the right and resulting in that grey chunk. So that's good now.

Issue 2 is that the tab panel content isn't showing below. That's just an issue on the storybook pages, though. These tabs were set to "auto" which means it should switch between the "fixed" and "compact" sizes based on container queries. However, those queries don't seem to work correctly in the storybook. So the tabs are showing as "fixed", but the panels are using the smaller padding-top for the "compact" variant, meaning it's hidden behind the tabs here.

On storybook if you check glass for "fixed", "compact", or "full" it should appear correct. And I tested by putting the tabs component into a Midnights page, and the "auto" sizing with the glass worked correctly there.